Sorting_Array.sh

```diff
@@ -0,0 +1,35 @@
+declare -a MYARRAY
+MYARRAY=( "$@" )
```

Comment:
what this command is doing?

Reply:
Line 1: Declaring an array MYARRAY
Line 2: Storing all values passed using command line argument to MYARRAY.